api/envoy/extensions/filters/http/ip_tagging/v3/ip_tagging.proto

```diff
@@ -39,6 +40,22 @@ message IPTagging {
     EXTERNAL = 2;
   }
 
+  // Describes how to apply the tags to the headers.
+  enum HeaderAction {
+    // (DEFAULT) The header specified in :ref:`ip_tag_header <envoy_v3_api_field_extensions.filters.http.ip_tagging.v3.IPTagging.ip_tag_header>`
+    // will be dropped, before the tags are applied. Sanitize will be run regardless of if the source is a trusted IP or not.
+    //
+    // Note that the header will be visible unsanitized to any filters downstream of the ip-tag-header filter, unless it has an *x-envoy* prefix.
```

Comment:
The word "downstream" may be ambiguous here.
I'm assuming it implies the filters that are invoked after this filter, but I'm not sure. Can you clarify it (probably by not using the word "downstream")?